test/raven.utils.js

```diff
@@ -121,6 +121,29 @@ describe('raven.utils', function () {
       }
     });
 
+    it('should trim long source line in surrounding source context', function (done) {
+      var parseStack = raven.utils.parseStack;
+      var callback = function (frames) {
+        var frame = frames.pop();
+
+        frame.pre_context.should.be.an.instanceOf(Array);
+        frame.pre_context[frame.pre_context.length - 1].trim().should.endWith('{snip}');
+        frame.context_line.should.be.type('string');
+        frame.context_line.trim().should.startWith('{snip}');
+        frame.context_line.trim().should.endWith('{snip}');
+        frame.post_context.should.be.an.instanceOf(Array);
+        frame.post_context[0].trim().should.endWith('{snip}');
```

Comment:
I feel it's not sufficient to just test for presence of `{snip}` – can we improve this test to also assert the content of the snipped code is as expected (e.g. what appears between `{snip}`)?